testdata/README.md

```diff
@@ -2,6 +2,24 @@
 
 This directory contains cryptographic artifacts necessary for the Bootz implementation. The files here are included as examples only and real implementations would need their own method of generating, storing and retrieving these.
 
-## ca.pem
+## Regenerating test data
 
-This is an x509 certificate that represents the device manufacturer's root (or some intermediate) CA. It is required that the ownership voucher passed from the Bootz server is signed by this CA.
\ No newline at end of file
+All certificates, private keys and OVs can be regenerated by running the `generate` binary. Ensure all text files are deleted from this directory before running to avoid conflicts.
+
+The current files in this directory were generated using the command `./generate -vendor "Cisco" -owner "Google" -serials "123A,123B"`.
+
+### vendorca_{pub|priv}.pem
+
+This is an x509 certificate/RSA keypair that represents the device manufacturer's root (or some intermediate) CA. This is used to sign the OVs.
```

Comment:
I think we should mention that self-signed certificates are used for convenience, but in production cases a reliable CA chain should be used instead.